src/simplecycles.jl

```diff
@@ -0,0 +1,57 @@
+using Combinatorics: powerset
+using Graphs: Graphs, is_tree, simplecycles_limited_length
+using NamedGraphs.GraphsExtensions: add_edges, disjoint_union, edge_subgraph
+using SplitApplyCombine: group
+
+function Graphs.simplecycles_limited_length(g::AbstractNamedGraph, max_cycle_size::Int64)
+  vs = collect(vertices(g))
+  cycles = simplecycles_limited_length(position_graph(g), max_cycle_size)
+  cycles = [[vs[i] for i in cycle] for cycle in cycles]
```

Comment:
```suggestion
  cycles = map(cycle -> map(i -> vs[i], cycle), cycles)
```
Mostly a style thing, but I prefer using `map` (it is a bit more generic since in general it can preserve the type of the input collection).